README.md

```diff
@@ -847,6 +866,30 @@ __AWESOME__ apps counter: __150__ 🎉
 - [x] [GitHub](https://github.com/oxen-io/session-android)
 - [x] [Official page](https://getsession.org/)
 
+### Nekogram X
```

Comment:
So you use Telegram and used each of these apps, and they are good?

Reply:
It does support however as I remember - not by default.
Yeah I saw lots of invite links to telegram groups.

I see. Why not include the original Telegram then?

Reply:
I see.
Yeah.